src/mpi/datatype/looputil.c

```diff
@@ -365,12 +365,17 @@ int MPIR_Segment_init(const void *buf,
     return 0;
 }
 
-/* Segment_alloc
+/* MPIR_Segment_alloc
  *
```

Comment:
This change looks like a pure comment fix, which is unrelated to this commit.

Reply:
Done.